crypto/mbedtls/Kconfig

```diff
@@ -0,0 +1,25 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+menuconfig CRYPTO_MBEDTLS
+	bool "Mbed TLS Cryptography Library"
+	default n
+	---help---
+		Enable support for Mbed TLS.
+
+if CRYPTO_MBEDTLS
+
+config MBEDTLS_VERSION
+	string "MBEDTLS Version"
+	default "2.25.0"
+
+config MBEDTLS_DEFAULT_TASK_STACKSIZE
+	int "Mbed TLS app default stack size"
+	default 8192
```

Comment:
can we use the small stack size or DEFAULT_TASK_STACKSIZE?

Reply:
The issue is the default stack size is only 2KB but I normally see 3-4KB on real hardware.  I get that the sim is probably much higher so if we kept this I could conditional this like the DEFAULT_TASK_STACKSIZE.  Thoughts?

Reply:
Yes, sim may need more stack size, DEFAULT_TASK_STACKSIZE has special value(64KB?) for sim.

Reply:
So what are you thinking is the right thing for some of these apps?  DEFAULT_TASK_STACKSIZE wont work for most of them.  The an especially bad offender is the app for testing parsing CRL it allocates a 100KB buffer on the stack right away (I don't know why they don't use the heap for this...)
https://github.com/ARMmbed/mbedtls/blob/6fbff5b557efe661cc019ef59f42c835524e9bf2/programs/x509/crl_app.c#L75

Reply:
Why not create PRfor mbedtls to allocate 100KB from heap:)